src/getCommitMeaning.ts

```diff
@@ -1,4 +1,7 @@
-import conventionalCommitsParser from "conventional-commits-parser";
+import {
+	parser,
+	toConventionalChangelogFormat,
+} from "@conventional-commits/parser";
```

Comment:
[Refactor] 🤔 `@conventional-commits/parser` hasn't had a commit or sustained GitHub issue triage in 3 years, and uses `throw` as a form of control flow. Whereas `conventional-commits-parser` released 3 months ago. I'm a little averse to going down to an older, less-maintained package if it's avoidable.

What do you think about instead using a small regex to check for a subject, optionally an area/scope, and `!` at the beginning? `/^\w+(\(\w+\))?!:/` or something like that?